mediator/med_phases_ocnalb_mod.F90

```diff
@@ -449,9 +470,15 @@ subroutine med_phases_ocnalb_run(gcomp, rc)
              rlon = const_deg2rad * ocnalb%lons(n)
              cosz = shr_orb_cosz( nextsw_cday, rlat, rlon, delta )
              if (cosz  >  0.0_r8) then !--- sun hit --
-                ocnalb%anidr(n) = (.026_r8/(cosz**1.7_r8 + 0.065_r8)) +   &
-                                  (.150_r8*(cosz         - 0.100_r8 ) *   &
-                                  (cosz - 0.500_r8 ) * (cosz - 1.000_r8 )  )
+                if (use_briegleb) then
```

Comment:
@matsbn - looking at the original code - this should be `if (.not use_briegleb) ` not `if (use_briegleb)`.
Looking at the original code it seems that the if blocks are not correct.